### PR TITLE
TagPicker: update styles to match toolkit

### LINKED ISSUE #9899: TagPicker: Fluent theme
Updating TagPicker to most current web fluent toolkit.

### PR DESCRIPTION
**Pull request by maxwellred**
#### Pull request checklist

- [] Addresses an existing issue: Fixes #9899 
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Update styles to match toolkit
Added screener tests
Added RTL to the clearbutton rounded corners so they switch sides correctly.

#### Screenshots

Selected TagItem clear button -pressed state
BEFORE
Screen Shot 2019-08-19 at 10 07 30 PM

AFTER
Screen Shot 2019-08-19 at 10 09 08 PM

RTL added to clear button (in order to alternate which corners get rounded 2px)
BEFORE
Screen Shot 2019-08-19 at 10 22 54 PM

AFTER
Screen Shot 2019-08-19 at 10 23 26 PM

Read Only state (REMOVED from this PR and will be added in later PR)
Screen Shot 2019-08-19 at 10 24 52 PM

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/10200)

### CODE REVIEW

**Comment by maxwellred** [comment]
Closing and opening to RE-run Screener test regarding the unrelated change observed in Callout

File: packages/office-ui-fabric-react/src/components/pickers/BasePicker.types.ts [hunk 1]
```diff
@@ -202,6 +202,10 @@ export interface IBasePickerProps<T> extends React.Props<any> {
    * Theme provided by styled() function.
    */
   theme?: ITheme;
+  /**
+   * Optional read only state applied to tag picker. Removes the clear button from the TagItem pill.
+   */
+  readOnly?: boolean;
```
Comment (Vitalius1) [inline packages/office-ui-fabric-react/src/components/pickers/BasePicker.types.ts]: Not sure we should add this prop on the `IBasePickerProps` which is supposed to be generic and it only pipes to a Tag picker. @joschect where would be the best place to add this prop so it only pipes for `TagPicker` scenario?

Comment (joschect) [reply packages/office-ui-fabric-react/src/components/pickers/BasePicker.types.ts]: I don't actually think we can add readonly as is. I'm pretty sure you'd still be able to delete each item with just backspace. Is readonly just supposed to remove the X? That should be easily achieved with a different Tag item and different styles.

Comment (maxwellred) [reply packages/office-ui-fabric-react/src/components/pickers/BasePicker.types.ts]: Per @joschect and my conversation I'll be removing the readOnly ability and add that functionality in a later PR

**Comment by maxwellred** [comment]
Hi @betrue-final-final I was hoping you could take a look at the new Screener changes which are the same as before but I removed the read only as I'll be working on that feature implementation in a later PR, thanks!

**Comment by micahgodbolt** [comment]
screener is now passing. are we just waiting for codeowners? @joschect ?

**Pull request merged**